server/routes/api/ads.js

```diff
@@ -109,11 +109,11 @@ router.post('/', async (req, res) => {
     try {
         await db.query('BEGIN');
```

Comment:
I know it's not your code, but why?

Reply:
The purpose of this line it to create a transaction (you can't send 2 SQL statements to the pool)